backend/open_webui/config.py

```diff
@@ -931,6 +932,9 @@ class BannerModel(BaseModel):
 
 VECTOR_DB = os.environ.get("VECTOR_DB", "chroma")
 
+if VECTOR_DB == 'pgvector' and not DATABASE_URL.startswith("postgres"):
```

Comment:
We should have a separate `PGVECTOR_DB_URL` env var, which takes in DATABASE_URL as it's default value if PGVECTOR_DB_URL env var is "" or None.